files/BatteryTools.py

```diff
@@ -4,7 +4,7 @@
 import PySAM.BatteryStateful as BattStfl
 
 
-def battery_model_sizing(model, desired_power, desired_capacity, desired_voltage, size_by_ac_not_dc=None):
+def battery_model_sizing(model, desired_power, desired_capacity, desired_voltage, size_by_ac_not_dc=None, module_dict=None):
     """
     Sizes the battery model using its current configuration such as chemistry, cell properties, etc
     and modifies the model's power, capacity and voltage without changing its fundamental properties
```

Comment:
`battery_model_sizing` is the entry-point for the user, so perhaps you could add a bit more description about what the `module_dict` is for, such as:

"The battery's thermal parameters (surface area and mass) are modified according to assumptions about the mass and volume per specific energy for a single battery module. These "default" assumptions are built-in and chemistry dependent. If the battery's thermal parameters should be sized according to a particular module's capacity and surface area, use the `module_dict` input.